### PR TITLE
Add EKS-A support details to docs

### PR DESCRIPTION
**Pull request by chrisnegus**
*Description of changes:*
Adding information about supported Kubernetes versions and operating systems, as well as support FAQs from PM, to docs Support page.

### CODE REVIEW

**Comment by chrisnegus** [comment]
I removed the hold. Awaiting final reviews.

File: docs/content/en/docs/reference/support/support-scope.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Please do not create a public GitHub issue for security problems.
+
+## Operating system support
+
+EKS Anywhere has some level of support for the following operating system nodes:
+
+* **Bottlerocket**: Bottlerocket is the only fully supported operating system for EKS Anywhere nodes.
+Bottlerocket OVAs and images are distributed by the EKS Anywhere project.
+See the [Artifacts]({{< relref "../artifacts" >}}) page for details.
+
```
Comment (TerryHowe) [inline docs/content/en/docs/reference/support/support-scope.md:24]: Need to add rhel here?

File: docs/content/en/docs/reference/support/support-versions.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Kubernetes support
+
+Each EKS Anywhere version generally includes support for multiple Kubernetes versions, with the exception of the initial few releases.
+Starting from EKS Anywhere version 0.11, the latest version supports at least four recent versions of Kubernetes.
+The end of support date of a Kubernetes version aligns with Amazon EKS in AWS as documented on the [Amazon EKS Kubernetes release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar). 
+
+Common vulnerabilities and exposures (CVE) patches and bug fixes, including those for the supported Kubernetes versions, are back-ported to the latest EKS Anywhere version (version n) and the previous one version (version n-1).
+The following table shows EKS Anywhere version support for different Kubernetes versions:
+
```
Comment (TerryHowe) [inline docs/content/en/docs/reference/support/support-versions.md:20]: this list will need updating soon

File: docs/content/en/docs/reference/support/support-versions.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+| EKS Anywhere version      | Kubernetes versions included | EKS Anywhere Release Date | CVE patches and bug fixes back-ported? |
+|------------|---------------------|---------------------------------|-------------------------|
+| 0.11 | 1.23, 1.22, 1.21, 1.20 | August 18, 2022 | Yes | 
+| 0.10 | 1.22, 1.21, 1.20 | June 30, 2022 | Yes | 
+| 0.9 | 1.21, 1.20 | May 12, 2022 | No | 
+| 0.8 | 1.22, 1.21, 1.20 | March 31, 2022 | No | 
+| 0.7 | 1.21, 1.20 | January 27, 2022 | No | 
+| 0.6 | 1.21, 1.20 | October 29, 2021 | No | 
+| 0.5 | 1.21, 1.20 | September 8, 2021 | No |
```
Comment (TerryHowe) [inline docs/content/en/docs/reference/support/support-versions.md:40]: what is our support commitment on these old releases?

**Review by TerryHowe — APPROVED** [review]

File: docs/content/en/docs/reference/support/support-scope.md (hunk 1 above)
Comment (elylee) [reply docs/content/en/docs/reference/support/support-scope.md:24]: Yes, @chrisnegus

File: docs/content/en/docs/reference/support/support-versions.md (hunk 2 above)
Comment (elylee) [reply docs/content/en/docs/reference/support/support-versions.md:20]: Yes, this is meant to be an evolving doc page.

File: docs/content/en/docs/reference/support/support-versions.md (hunk 3 above)
Comment (elylee) [reply docs/content/en/docs/reference/support/support-versions.md:40]: TBD. Jackson and I are still thinking about it. We don't have to make any public statements yet.

File: docs/content/en/docs/reference/support/support-versions.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+To see supported versions of Kubernetes for each release of EKS Anywhere, and information about that support, refer to the content below.
+
+
+## Kubernetes support
+
+Each EKS Anywhere version generally includes support for multiple Kubernetes versions, with the exception of the initial few releases.
+Starting from EKS Anywhere version 0.11, the latest version supports at least four recent versions of Kubernetes.
+The end of support date of a Kubernetes version aligns with Amazon EKS in AWS as documented on the [Amazon EKS Kubernetes release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar). 
+
+Common vulnerabilities and exposures (CVE) patches and bug fixes, including those for the supported Kubernetes versions, are back-ported to the latest EKS Anywhere version (version n) and the previous one version (version n-1).
```
Comment (elylee) [inline docs/content/en/docs/reference/support/support-versions.md:18]: Discussed with Jackson. Remove the last portion of the sentence where it says "...and the previous one version (version n-1)". For now, we will publicly say we only back-port fixes to version N.

File: docs/content/en/docs/reference/support/support-scope.md (hunk 1 above)
Comment (chrisnegus) [reply docs/content/en/docs/reference/support/support-scope.md:24]: Okay. I added a section on support for RHEL-based images.

File: docs/content/en/docs/reference/support/support-versions.md (hunk 4 above)
Comment (chrisnegus) [reply docs/content/en/docs/reference/support/support-versions.md:18]: Done.

**Review by TerryHowe — APPROVED** [review]

**Pull request merged**